go/parquet/pqarrow/file_writer.go

```diff
@@ -134,6 +134,13 @@ func (fw *FileWriter) RowGroupTotalBytesWritten() int64 {
 	return 0
 }
 
+// WriteBuffered allows to write records and decide where to break your row group
+// based on the TotalBytesWritten rather than on the max row group len.
```

Comment:
TotalBytesWritten will still break row groups on the max row group len, not TotalBytesWritten. It does, however, allow users to use TotalBytesWritten to decide when to break row groups themselves by checking `TotalBytesWritten` and calling `NewBuffereredRowGroup` themselves.

Reply:
Done in 9c4ca94137a143986e69feb07a0ab94baee7e334